mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/spi/PojoMassIndexerAgent.java

```diff
@@ -24,6 +24,7 @@ static PojoMassIndexerAgent noOp() {
 	 * @return A future that completes successfully when other agents have been successfully suspended.
 	 * If no agents can be suspended (e.g. no coordination), returns a successfully completed future immediately.
 	 */
+	// TODO: Yoann, feels like we are talking about outbox-polling here that should work only with ORM? drop the example?
```

Comment:
IMO we can accept talking about concepts that only make sense for ORM if it helps with implementation of those SPIs.

Regardless, you could just replace "(e.g. automatic indexing)" with "(e.g. indexing plans)", since explicit indexing through indexing plan would also be suspended.